src/templates/components/image/index.js

```diff
@@ -17,8 +18,9 @@ const Image = ({
     alt,
     className,
     decoding = 'async',
-    imgClassName,
+    fetchPriority = 'auto',
```

Comment:
My only question would be whether we want to default to auto or null (no attribute)? I don't have a preference.

Reply:
Agreed. I will change the default to null and undate the docs.